tests/integrationv2/providers.py

```diff
@@ -543,3 +576,188 @@ def setup_client(self):
+            Ciphers.DHE_RSA_AES128_SHA256:      "DHE-RSA:+AES-128-CBC:+SHA256",
+            Ciphers.DHE_RSA_AES256_SHA256:      "DHE-RSA:+AES-256-CBC:+SHA256",
+            Ciphers.DHE_RSA_AES128_GCM_SHA256:  "DHE-RSA:+AES-128-GCM:+AEAD",
+            Ciphers.DHE_RSA_AES256_GCM_SHA384:  "DHE-RSA:+AES-256-GCM:+AEAD",
+            Ciphers.DHE_RSA_CHACHA20_POLY1305:  "DHE-RSA:+CHACHA20-POLY1305:+AEAD",
+
+            Ciphers.AES128_SHA:         "RSA:+AES-128-CBC:+SHA1",
+            Ciphers.AES256_SHA:         "RSA:+AES-256-CBC:+SHA1",
+            Ciphers.AES128_SHA256:      "RSA:+AES-128-CBC:+SHA256",
+            Ciphers.AES256_SHA256:      "RSA:+AES-256-CBC:+SHA256",
+            Ciphers.AES128_GCM_SHA256:  "RSA:+AES-128-GCM:+AEAD",
+            Ciphers.AES256_GCM_SHA384:  "RSA:+AES-256-GCM:+AEAD",
+
+            Ciphers.ECDHE_ECDSA_AES128_SHA:         "ECDHE-ECDSA:+AES-128-CBC:+SHA1",
+            Ciphers.ECDHE_ECDSA_AES256_SHA:         "ECDHE-ECDSA:+AES-256-CBC:+SHA1",
+            Ciphers.ECDHE_ECDSA_AES128_SHA256:      "ECDHE-ECDSA:+AES-128-CBC:+SHA256",
+            Ciphers.ECDHE_ECDSA_AES256_SHA384:      "ECDHE-ECDSA:+AES-256-CBC:+SHA384",
+            Ciphers.ECDHE_ECDSA_AES128_GCM_SHA256:  "ECDHE-ECDSA:+AES-128-GCM:+AEAD",
+            Ciphers.ECDHE_ECDSA_AES256_GCM_SHA384:  "ECDHE-ECDSA:+AES-256-GCM:+AEAD",
+
+            Ciphers.ECDHE_RSA_AES128_SHA:           "ECDHE-RSA:+AES-128-CBC:+SHA1",
+            Ciphers.ECDHE_RSA_AES256_SHA:           "ECDHE-RSA:+AES-256-CBC:+SHA1",
+            Ciphers.ECDHE_RSA_AES128_SHA256:        "ECDHE-RSA:+AES-128-CBC:+SHA256",
+            Ciphers.ECDHE_RSA_AES256_SHA384:        "ECDHE-RSA:+AES-256-CBC:+SHA384",
+            Ciphers.ECDHE_RSA_AES128_GCM_SHA256:    "ECDHE-RSA:+AES-128-GCM:+AEAD",
+            Ciphers.ECDHE_RSA_AES256_GCM_SHA384:    "ECDHE-RSA:+AES-256-GCM:+AEAD",
+            Ciphers.ECDHE_RSA_CHACHA20_POLY1305:    "ECDHE-RSA:+CHACHA20-POLY1305:+AEAD"
+        }.get(cipher)
+
+    @staticmethod
+    def protocol_to_priority_str(protocol):
+        return {
+            Protocols.TLS10.value: "VERS-TLS1.0",
+            Protocols.TLS11.value: "VERS-TLS1.1",
+            Protocols.TLS12.value: "VERS-TLS1.2",
+            Protocols.TLS13.value: "VERS-TLS1.3"
+        }.get(protocol.value)
+
+    @staticmethod
+    def curve_to_priority_str(curve):
+        return {
+            Curves.P256:    "CURVE-SECP256R1",
+            Curves.P384:    "CURVE-SECP384R1",
+            Curves.P521:    "CURVE-SECP521R1",
+            Curves.X25519:  "CURVE-X25519"
+        }.get(curve)
+
+    @staticmethod
+    def sigalg_to_priority_str(sigalg):
+        return {
+            Signatures.RSA_SHA1:    "SIGN-RSA-SHA1",
+            Signatures.RSA_SHA256:  "SIGN-RSA-SHA256",
+            Signatures.RSA_SHA384:  "SIGN-RSA-SHA384",
+            Signatures.RSA_SHA512:  "SIGN-RSA-SHA512",
+
+            # GnuTLS only supports this signature in TLS 1.1
+            # Signatures.RSA_SHA224:  "SIGN-RSA-SHA224",
```

Comment:
Maybe an overall summary comment here about where these came from?

Reply:
I added these comments because the other tests were using these signature algorithms. However, the v1 tests don't use them, so I just removed the comments for now: 5f221b0a1b44c1d71b72dc51abe672bc3070576a